src/Sarif.Multitool/RebaseUriCommand.cs

```diff
@@ -37,7 +37,11 @@ public int Run(RebaseUriOptions rebaseOptions)
 
                 var sarifFiles = GetSarifFiles(rebaseOptions);
 
-                Directory.CreateDirectory(rebaseOptions.OutputFolderPath);
+                if (!rebaseOptions.Inline)
```

Comment:
Can the mutual exclusivity be expressed high-up? Say the at the commandOptions level or at least as an assert for this command?